nixos/modules/services/misc/redlib.nix

```diff
@@ -88,8 +87,7 @@ in
     };
   };
 
-  config = mkIf cfg.enable {
-    systemd.packages = [ cfg.package ];
```

Comment:
Here's another rebase error, I'm assuming.

Reply:
removed the commit touching that file. already resolved via e286b91ebca6f7de81cefb87492440582a565990